tutorial/functions.py

```diff
@@ -17,18 +17,18 @@ def __init__(self, title=""):
         q2 = Question(
             question="What is the purpose of type hints in Python function definitions?",
             options={
-                "To enforce strict typing": "",
-                "To make the code more readable": "",
-                "To document the expected types": "Correct!",
+                "To enforce strict typing": "Does the interpreter enforce type hints?",
+                "Only to make the code more readable": "Not only...",
+                "To document the expected types of function arguments and return values": "Correct!",
             },
-            correct_answer="To document the expected types",
+            correct_answer="To document the expected types of function arguments and return values",
```

Comment:
The code readability also must be a part of the correct answer. Otherwise, it should not be mentioned in the advantages of type hints in the text above.

Reply:
Another option is to rephrase the question so that the answer fits it.

Reply:
sure, we can do it as a follow-up PR.